src/main/sd/command.c

```diff
@@ -0,0 +1,72 @@
+#include "sd.h"
+#include "spi.h"
+#include "crc.h"
+#include "command.h"
+
+#include "debug.h"
+#include "pin.h"
+
+#include <stdlib.h>
+#include <string.h>
+
+bool wait_for_response(uint8_t* buffer)
```

Comment:
Maybe use(so we can use different error codes in a future): 
```suggestion
uint8_t wait_for_response(uint8_t* buffer)
```